packages/firebase_auth/firebase_auth/lib/firebase_auth.dart

```diff
@@ -6,19 +6,32 @@ library firebase_auth;
 
 import 'dart:async';
 
+import 'package:firebase_auth_platform_interface/firebase_auth_platform_interface.dart';
 import 'package:firebase_core/firebase_core.dart';
-import 'package:flutter/services.dart';
 import 'package:meta/meta.dart';
 
+export 'package:firebase_auth_platform_interface/firebase_auth_platform_interface.dart'
+    show
+        AuthCredential,
+        AuthException,
+        EmailAuthCredential,
+        FacebookAuthCredential,
+        GithubAuthCredential,
+        GoogleAuthCredential,
+        PhoneAuthCredential,
+        TwitterAuthCredential,
+        PhoneVerificationCompleted,
+        PhoneVerificationFailed,
+        PhoneCodeSent,
+        PhoneCodeAutoRetrievalTimeout;
```

Comment:
Is there anything that we're exporting from the platform_interface that we do NOT want to reexport?

(Maybe it's shorter to `export 'blah' hide FooBar;`?)

Reply:
(Outdated, resolving)